server/chat-commands/core.ts

```diff
@@ -484,6 +484,13 @@ export const commands: ChatCommands = {
 		user.resetName();
 	},
 
+	nfb: 'nofeedback',
+	nofeedback(target, room, user) {
+		if (!target?.startsWith('/')) return this.parse('/help nofeedback');
```

Comment:
```suggestion
		if (!target.startsWith('/')) return this.parse('/help nofeedback');
```

(`target` isn't nullable)